Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -819,4 +819,7 @@ def SwapKeepAndDrop(l):
 if fastSim.isChosen():
     for _entry in [FEVTDEBUGHLTEventContent,FEVTDEBUGEventContent,RECOSIMEventContent,AODSIMEventContent,RAWAODSIMEventContent]:
         fastSimEC.dropSimDigis(_entry.outputCommands)
-    
+
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+for _entry in [FEVTDEBUGEventContent,FEVTDEBUGHLTEventContent,FEVTEventContent]:
+  phase2_tracker.toModify(_entry, outputCommands = _entry.outputCommands + ['keep Phase2TrackerDigiedmDetSetVector_*_*_*'])
```

Comment:
bug/issue discovery is often incidental.
So, it's found here and not earlier.
If you know the list of modules making Phase2TrackerDigiedmDetSetVector, please change it to match that list.